misc/fuzzer_config.txt

```diff
@@ -0,0 +1,55 @@
+use_parent_methods: true
+add_to_tree: true
+delay_removing_added_nodes_to_next_frame: true
+test_one_class_multiple_times: true
+
+custom_classes:
+FastNoiseLite
```

Comment:
Note, we could use `config.py` which has a function returning the list of all classes available on the documentation